### PR TITLE
Single Test Case for default layout options of Sidebar

### PR DESCRIPTION
**Pull request by bsfsnehal**
### Description

Test case for Default layout of sidebar
### How has this been tested?

Branch-Name :- sidebar
Test case Execution :- npm run test:e2e:interactive — tests/e2e/specs/customizer/sidebar/def-layout-sidebar.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/sidebar/def-layout-sidebar.test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../utils/set-customize';
+    describe('Default layout for Sidebar setting in customizer', () => {
+        it( 'Default layout for sidebar from customizer should apply corectly', async () => {
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/sidebar/def-layout-sidebar.test.js:4]: spelling mistake

File: tests/e2e/specs/customizer/sidebar/def-layout-sidebar.test.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            await page.waitForSelector( '.secondary' );
+
+            await expect( {
+                selector: '.secondary',
+                property: '',
+            } ).cssValueToBe( `` );  
+        });
+     
+
+        it( 'Default layout for sidebar  as No Sidebar should apply corectly', async () => {
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/sidebar/def-layout-sidebar.test.js:24]: more than 1 spaces bet 2 words

File: tests/e2e/specs/customizer/sidebar/def-layout-sidebar.test.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            'site-sidebar-layout': 'left-sidebar',
+
+            };
+            await setCustomize( leftSidebar );
+
+            await page.goto( createURL( '/' ), {
+                waitUntil: 'networkidle0',
+            } );
+
+            await page.waitForSelector( '.secondary' );
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/sidebar/def-layout-sidebar.test.js:15]: can we wait for some sidebar specific class?

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/sidebar/def-layout-sidebar.test.js:15]: Means?

**Review by sushmak02 — CHANGES_REQUESTED** [review]
Review the changes suggested and get them done at all the places

File: tests/e2e/specs/customizer/sidebar/def-layout-sidebar.test.js (hunk 3 above)
Comment (sushmak02) [reply tests/e2e/specs/customizer/sidebar/def-layout-sidebar.test.js:15]: This class is not specifically related to left-sidebar. "secondary"

Comment (sushmak02) [reply tests/e2e/specs/customizer/sidebar/def-layout-sidebar.test.js:15]: await page.waitForSelector( '.ast-right-sidebar #secondary' );
expecting something like this

**Review by sushmak02 — APPROVED** [review]

**Review by sushmak02 — APPROVED** [review]

**Pull request closed without merging**